.github/workflows/test-package.yml

```diff
@@ -216,3 +248,40 @@ jobs:
           github-token: ${{ secrets.github_token }}
           parallel-finished: true
 
+  run_pdfbox_example_linux:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./jni_gen/examples/pdfbox_example
+    steps:
+      - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+          cache: true
+          cache-key: 'flutter-:os:-:channel:-:version:-:arch:-:hash:'
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+      - run: |
+          sudo apt-get update -y
+          sudo apt-get install -y ninja-build libgtk-3-dev
+      - run: flutter config --enable-linux-desktop
+      - run: flutter pub get
+      - name: Generate bindings
+        run: |
+          flutter pub get
+          flutter pub run jni_gen --config jnigen.yaml
+        working-directory: ./jni_gen/examples/pdfbox_example/pdfbox_plugin
+      - name: Analyze generated bindings
+        run: flutter analyze
+      - name: Build flutter example for pdfbox_plugin
+        run: flutter build linux
+      - name: Run standalone example
+        run: |
+          dart pub get
+          dart run jni:setup && dart run jni:setup -p pdfbox_plugin
+          wget 'https://dart.dev/guides/language/specifications/DartLangSpec-v2.2.pdf'
```

Comment:
@eernstg The spec is serving many purposes! 🚀 (In this case testing our Java interop by using a Java PDF reading library. 😄)

Next step, make Dart sentient, so that it can actually read the contents of its own spec. 😆 🤯

Reply:
:smile: